### PR TITLE
Check for existing assistant when login to cloud

### PR DESCRIPTION
**Pull request by piitaya**
## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/cloud/login/cloud-login.ts [hunk 1]
```diff
@@ -214,19 +218,39 @@ export class CloudLogin extends LitElement {
     this._requestInProgress = true;
 
     try {
+      let pipelines: AssistPipeline[] = [];
+      let preferred_pipeline: string | null = null;
+      try {
+        const response = await listAssistPipelines(this.hass);
```
Comment (bramkragten) [inline src/panels/config/cloud/login/cloud-login.ts:224]: let's not await here, but after login

Comment (piitaya) [reply src/panels/config/cloud/login/cloud-login.ts:224]: We need to fetch pipelines before to know if it's new or not.

Comment (bramkragten) [reply src/panels/config/cloud/login/cloud-login.ts:224]: we can start the fetch, but don't have to await it

Comment (bramkragten) [reply src/panels/config/cloud/login/cloud-login.ts:224]:
```ts
const listPromise = listAssistPipelines(this.hass);
await login();
const response = await listPromise;
```

Comment (piitaya) [reply src/panels/config/cloud/login/cloud-login.ts:224]: Not sure about concurrent requests 😅 Listing pipelines should be faster than login + pipeline creation.

Comment (bramkragten) [reply src/panels/config/cloud/login/cloud-login.ts:224]: it is faster, so no need to wait before starting the login, as we don't need it before the login is finished 🙃

**Comment by bramkragten** [comment]
No longer needed after https://github.com/home-assistant/core/pull/92058

**Pull request closed without merging**